unittests.py

```
@@ -0,0 +1,31 @@
+    Quarter,
+    Month,
+    Week,
+    Day
+]
+START_DATE = date(1302, 7, 11)
+END_DATE = date(2019, 11, 25)
+
+class TimeUnitTest(unittest.TestCase):
+    def date_range_yield(self):
+        dd = (END_DATE - START_DATE).days
+        for i in range(dd):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)